RecoParticleFlow/Benchmark/src/PFJetBenchmark.cc

```diff
@@ -313,42 +314,7 @@ void PFJetBenchmark::process(const reco::PFJetCollection& pfJets, const reco::Ge
 	  //	    << " has no track ref.." << std::endl;
 	  continue;
 	}
-	unsigned int iter = 0; 
-	switch (trackRef->algo()) {
-	case TrackBase::ctf:
-        case TrackBase::duplicateMerge:
-	case TrackBase::initialStep:
-	  iter = 0;
-	  break;
-	case TrackBase::lowPtTripletStep:
-	  iter = 1;
-	  break;
-	case TrackBase::pixelPairStep:
-	  iter = 2;
-	  break;
-	case TrackBase::detachedTripletStep:
-	  iter = 3;
-	  break;
-	case TrackBase::mixedTripletStep:
-	  iter = 4;
-	  break;
-	case TrackBase::pixelLessStep:
-	  iter = 5;
-	  break;
-	case TrackBase::tobTecStep:
-	  iter = 6;
-	  break;
-	case TrackBase::conversionStep:
-	  iter = 7;
-	  //std::cout << "Warning in entry " << entry_ << " : iter = " << trackRef->algo() << std::endl;
-	  //std::cout << ic << " " << *(constituents[ic]) << std::endl;
-	  break;
-	default:
-	  iter = 8;
-	  std::cout << "Warning in entry " << entry_ << " : iter = " << trackRef->algo() << std::endl;
-	  std::cout << ic << " " << *(constituents[ic]) << std::endl;
-	  break;
-	}
+	unsigned int iter = trackRef->algo()>6 ? 6: trackRef->algo();
```

Comment:
Hi Matti,
This one I had not seen . In any case I cannot do anything about that since it is EGM code and they should take care of it [In fact I dont think it is running any more either this is why I wanted to have a clean up ]